packages/popover/src/theme.ts

```diff
@@ -20,4 +20,7 @@ export const defaultPopoverTheme: IPopoverTheme = {
       color: '#ddd',
     },
   },
+  wrapper: {
+    cursor: 'pointer',
```

Comment:
I am not sure about this solution only in one case. If for some reason popover width will be 200px (for example) and icon (or another component) inside will be 50px (for example) there will be 150px of space with hover.
But I think it is a rare case and there are ways to solve it.